### PR TITLE
Delete class command

### LINKED ISSUE #5: Delete tutorial class
As a TA, I can delete a student from my class if they drop the module/class.

### PR DESCRIPTION
**Pull request by whelan-low**
Fixes #5.

Adds the functionality to delete the module code: tutorial class specified from the model. If module code does not exist, it returns an error. If tutorial does not exist within the module, it brings up the list of tutorial classes that can be deleted from that module.

TODO:
- [x] add more test cases
- bring up list of modules when non-existent module code is entered (Need to do?)

### CODE REVIEW

File: src/main/java/seedu/address/model/module/ModuleCode.java [hunk 1]
```diff
@@ -136,4 +154,14 @@ public boolean hasTutorialClass(TutorialClass tutorialString) {
...
         tutorialClasses.add(tutorialString);
     }
+
+    /**
+     * Deletes a tutorial with the given name from the module.
+     * The tutorial has to exist to be used in this function.
+     *
+     * @param tutorialString name of tutorial class to be deleted.
+     */
+    public void deleteTutorialClass(TutorialClass tutorialString) {
```
Comment (Zack-Tay) [inline src/main/java/seedu/address/model/module/ModuleCode.java:164]: Is there a better naming here? I feel like tutorialString is okay if the type is a String, but in this case if is type TutorialClass that I think we can stick with the convention of tutorialClass.

File: src/main/java/seedu/address/model/module/ModuleCode.java [hunk 2]
```diff
@@ -128,6 +128,24 @@ public boolean hasTutorialClass(TutorialClass tutorialString) {
...
     }
 
+    /**
+     * List all the tutorial classes under this module.
+     *
+     * @return String of tutorial classes under this module.
+     */
+    public String listTutorialClasses() {
+        if (tutorialClasses.size() == 0) {
+            return String.format("Tutorials in %s: None!", value);
```
Comment (Zack-Tay) [inline src/main/java/seedu/address/model/module/ModuleCode.java:138]: Has this been tested on the application itself? Just wanna check that value here will not be null, or an empty string.

**Review by Zack-Tay — COMMENTED** [review]
LGTM! Just some minor nits! Thanks for the great work

File: src/main/java/seedu/address/model/module/ModuleCode.java (hunk 2 above)
Comment (whelan-low) [reply src/main/java/seedu/address/model/module/ModuleCode.java:138]: The value is the module code name itself, not related to the tutorial classes.

File: src/main/java/seedu/address/model/module/ModuleCode.java (hunk 1 above)
Comment (whelan-low) [reply src/main/java/seedu/address/model/module/ModuleCode.java:164]: Good point. I'll rename it

**Comment by whelan-low** [comment]
I have renamed the 'value' in ModuleCode and TutorialClass to moduleCode and tutorialName respectively.

**Review by Zack-Tay — APPROVED** [review]
LGTM!

**Pull request merged**